src/algebraic_geometry/AffineScheme.lean

```diff
@@ -385,7 +385,7 @@ begin
     exact congr_arg subtype.val (X.map_prime_spectrum_basic_open_of_affine x).symm }
 end
 
-lemma is_affine_open.exists_basic_open_subset {X : Scheme} {U : opens X.carrier}
+lemma is_affine_open.exists_basic_open_le {X : Scheme} {U : opens X.carrier}
```

Comment:
But the statement is about `subset`? So wasn't the old name more correct?

Reply:
I forgot to change the statement as well. Thanks for pointing out.